docs/cytomining-ecosystem/index.html

```diff
@@ -19,17 +19,23 @@ <h1>Cytomining Ecosystem Software Landscape Analysis</h1>
                 href="https://github.com/WayScience/software-landscape-analysis">https://github.com/WayScience/software-landscape-analysis</a>.
         </p>
 
-        <h2>Focused Report</h2>
-        <p>A focused report is provided below to help describe landscape elements related to
-            project user base size, usage, and maturity.</p>
+        <h2>Targeted Software Report</h2>
+        <p>A targeted report is provided below to illustrate specific project usage, growth, and dependency.</p>
         <ul>
-            <li><a href="report.html">Focused Report</a></li>
+            <li><a href="target-project-report.html">Targeted Software Report</a></li>
         </ul>
 
-        <h2>Data Profile</h2>
-        <p>A data profile (generated from <a
+        <h2>Software Landscape Report</h2>
+        <p>A landscape report is provided below to help describe projects related to
+           target ecosystem user base size, usage, and maturity.</p>
```

Comment:
```suggestion
        <p>We provide a landscape report below to help describe projects related to
           our ecosystem's user base size, usage, and maturity.</p>
```

I'm not sure if my edits retain your original meaning!

Reply:
Thank you for this, looks great!